### ISSUE TITLE
DateConverter is not stateless

### ISSUE THREAD

**Issue by Raeisifard** [eloyzone/jalali-calendar#3]
When I run the code retrieved from "Some basic code"  section in read.me file there is "1370-12-17" in  jalaliDate1 as value instead of "1370-11-28". It seems that "dateConverter" is one time usable and is not stateless. So after each use of "dateConverter.jalaliToGregorian()" I must instantiate it again!

**Comment by eloyzone** [comment]
Yes you are right and see [this issue](https://github.com/eloyzone/jalali-calendar/issues/2#issuecomment-813178938).

Although it is easy to be fixed, I am busy. I hope to make it stateless as soon as possible.

Thank you for mentioning.
